src/Models/RidersModel.php

```diff
@@ -65,4 +65,96 @@ public function getAllRiders(): array|false
 
         return $allRiders;
     }
+
+    public function getTeamIdFromTeam(string $team): int|false
+    {
+        $query = $this->db->prepare('SELECT `id` FROM `teams` WHERE `team` = :team;');
+        $query->execute(['team' => $team]);
+        $data = $query->fetch();
+
+        if ($data) {
+            return $data['id'];
+        }
+        return false;
+    }
+
+    public function getNationIdFromNationality(string $nationality): int|false
+    {
+        $query = $this->db->prepare('SELECT `id` FROM `nations` WHERE `nation` = :nation;');
+        $query->execute(['nation' => $nationality]);
+        $data = $query->fetch();
+
+        if ($data) {
+            return $data['id'];
+        }
+        return false;
+    }
+
+    public function addTeam(string $team): void
+    {
+        $query = $this->db->prepare('INSERT INTO `teams` (`team`) VALUES (:team);');
+        $query->execute(['team' => $team]);
+    }
+
+    public function addNation(string $nation): void
+    {
+        $query = $this->db->prepare('INSERT INTO `nations` (`nation`) VALUES (:nation);');
+        $query->execute(['nation' => $nation]);
+    }
+
+    public function addRider(
+        string $name,
+        string $image,
+        int $teamId,
+        int $nationId,
+        string $dob,
+        ?int $giroGcWins,
+        ?int $tourGcWins,
+        ?int $vueltaGcWins,
+        ?int $giroStageWins,
+        ?int $tourStageWins,
+        ?int $vueltaStageWins
+    ): void {
+        $query = $this->db->prepare("
+            INSERT INTO `riders` (
+            `name`,
+            `image`,
+            `team_id`,
+            `nation_id`,
+            `dob`,
+            `giro_gc`,
+            `tour_gc`,
+            `vuelta_gc`,
+            `giro_stages`,
+            `tour_stages`,
+            `vuelta_stages`
+            )
+            VALUES (
+            :name,
+            :image,
+            $teamId,
+            $nationId,
+            :dob,
+            :giroGcWins,
+            :tourGcWins,
+            :vueltaGcWins,
+            :giroStageWins,
+            :tourStageWins,
+            :vueltaStageWins
+            );
+        ");
+        $query->execute(
```

Comment:
You should make this method return execute - execute returns a bool to tell you whether or not the query worked. You could then use that when you call the method in addRider.php to display an error message if the query fails for whatever reason.